doc/man5/config.pod

```diff
@@ -27,6 +27,15 @@ The environment is mapped onto a section called B<ENV>.
 
 Comments can be included by preceding them with the B<#> character
 
+Other files can be included using the B<.include> directive followed
+by a path. If the path points to a directory all files with
```

Comment:
Customarily you'd have to specify an absolute path, right? But formally you can't prevent user from specifying relative path and making it work. Would it be appropriate to clarify that? That user would be responsible for making relative path work? And advice use of an absolute path otherwise?

BTW, here is related question, wouldn't user feel that it would be natural to specify relative path, but relative to position of the file containing the .include statement? Maybe too much? Though I for one feel that I might have appreciated that...

Reply:
I will clarify the documentation. As for implementing the 'natural' thing - path relative to the including file - this would be fairly non-trivial I think. Especially because the BIO that is containing the .include actually does not have to be a file at all. So I am sorry.

Reply:
No prob, it was just a wild idea... :-)